.travis.yml

```diff
@@ -1,50 +1,100 @@
+# Variables needed for this script are:
+
+# Case 1: Stored on Travis Settings
+  # For all environments:
+    # - DOCKER_CI_REPO_NAME
+    # - DOCKER_USERNAME
+    # - DOCKER_PASSWORD
+
+  #For dev environment:
+    # - AWS_ACCOUNT_USER_ID_DEV
+    # - AWS_ACCOUNT_SECRET_DEV
+    # - AWS_PROFILE_DEV <-- is this needed?
+
+  #For Prod environment:
+    # - AWS_ACCOUNT_USER_ID_PROD
+    # - AWS_ACCOUNT_SECRET_PROD
+    # - AWS_PROFILE_PROD <-- is this needed?
+
+  #For Demo environment:
+    # - AWS_ACCOUNT_USER_ID_DEMO
+    # - AWS_ACCOUNT_SECRET_DEMO
+    # - AWS_PROFILE_DEMO <-- is this needed?
+
+# Case 2: Created when Travis CI runs
+    # - IMAGE_TAG
+
 sudo: required
 
-language: node_js
-node_js:
-  - lts/*
+# language: node_js
+# node_js:
+#   - lts/*
+language: generic
+services:
+  - docker
+
+env:
+  global:
+    - COMMIT=${TRAVIS_COMMIT::7}
+    #Add IAMGE_TAG variable to build docker image
+    - IMAGE_TAG=${TRAVIS_COMMIT::7}
 
 before_install:
   # Removed command as pyenv is no longer found during deploy
   #- pyenv global 2.7.14  # For removing SNIMissingWarning warnings
-  - curl https://install.meteor.com | /bin/sh
-  - export PATH="$HOME/.meteor:$PATH"
+  # - curl https://install.meteor.com | /bin/sh
+  # - export PATH="$HOME/.meteor:$PATH"
```

Comment:
@tuananh1508 why do we need to comment these out? as far as I can tell, Meteor is needed in the machine so we need to make sure it's installed,